src/containers/Tenant/Diagnostics/TenantOverview/TenantOverview.scss

```diff
@@ -118,4 +118,19 @@
             cursor: pointer;
         }
     }
+
+    &__info {
+        margin-bottom: 36px;
+    }
+
+    &__cell-with-popover-wrapper {
+        display: flex;
+    }
+
+    &__cell-with-popover {
+        overflow: hidden;
+
+        white-space: nowrap;
+        text-overflow: ellipsis;
+    }
```

Comment:
Can you please rebase this branch and replace this styles with `CellWithPopover` component?

Reply:
If we replace custom popover wrapper in `TopTables` with `CellWithPopover`, then we will have different color between link and `...`. If its ok I can use `CellWithPopover` there

<img width="261" alt="image" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/45696255/beb2fe81-51a9-4721-980c-02f99e9cd601">

Reply:
We currently have such behaviour for `Nodes`. Is it possible to change it everywhere within `CellWithPopover`? If no, probably white dots will be fine

<img width="480" alt="Screen Shot 2023-10-23 at 14 16 47" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/23ee946c-dbc6-40fb-8acc-0c48cf26e348">